stl/src/vector_algorithms.cpp

```diff
@@ -1327,6 +1399,144 @@ namespace {
         }
     }
 
+    // _Minmax has exactly the same signature as the extern "C" functions
+    // (__std_min_Nn, __std_max_Nn, __std_minmax_Nn), up to calling convention.
+    // This makes sure the template specialization is fused with the extern "C" function.
+    // In optimized builds it avoids an extra call, as this function is too large to inline.
+    template <_Min_max_mode _Mode, class _Traits, const bool _Sign>
+    auto __stdcall _Minmax(const void* _First, const void* const _Last) noexcept {
+        using _Ty = std::conditional_t<_Sign, typename _Traits::_Signed_t, typename _Traits::_Unsigned_t>;
+
+        _Ty _Cur_min_val; // initialized in both of branches below
+        _Ty _Cur_max_val; // initialized in both of branches below
+
+        // We don't have unsigned 64-bit stuff, so will use sign correction just for that case

```

Comment:
Grammar: This might technically work (binding "We" to "will"), but "so we'll use" would sound much more natural.